app/workers/publishing_api_notifier.rb

```diff
@@ -5,8 +5,8 @@ def perform(edition_id, update_type = "normal")
     edition = Edition.find(edition_id)
```

Comment:
This class probably needs to be renamed as it is now only responsible for calling the publish endpoint. Maybe PublishingAPIPublisher?